map/routing_manager.hpp

```diff
@@ -259,6 +259,12 @@ class RoutingManager final
   {
     m_routingSession.SetTurnNotificationsUnits(units);
   }
+
+  void SetIsLandscape(bool isLandscape)
```

Comment:
Is this function needed? `m_routingManager.RoutingSession().SetIsLandscape()` should be enough, right?